base/reducedim.jl

```diff
@@ -574,14 +577,22 @@ any!(r, A)
 for (fname, op) in [(:sum, :+), (:prod, :*),
                     (:maximum, :scalarmax), (:minimum, :scalarmin),
                     (:all, :&), (:any, :|)]
+    function compose_pss(x)
```

Comment:
only used once, abbreviating the name isn't that helpful

Reply:
makes sense, wrote it in full